topic/machine-learning/automl/automl_classification_with_pycaret.py

```diff
@@ -64,7 +64,6 @@ def try_ensemble_model(model):
     blend_models(estimator_list=tuned_models)
     best_model = automl(optimize="AUC")
 
-    evaluate_model(best_model)
     final_model = finalize_model(best_model)
```

Comment:
Is it correct that this line has been removed within the other notebook?

Reply:
Yes, this is ipywidget which looks and behaves extremely bad in non notebook environments.
I hope you don't mind, that I forward-fixed this thing in an actually unrelated file.